internal/pkg/describe/describe.go

```diff
@@ -62,6 +62,18 @@ func flattenEnvVars(envName string, m map[string]string) []*EnvVars {
 	return envVarList
 }
 
+func flattenSecrets(envName string, m map[string]string) []*Secrets {
+	var secretsList []*Secrets
```

Comment:
nit: no need to include the type in the variable name since it's already explicit with the type
```suggestion
	var secrets []*Secret
```

Reply:
done